.github/workflows/builder_bazel_slsa3.yml

```diff
@@ -76,6 +81,19 @@ on:
           When run on other triggers, attestations are signed and have an "intoto.sigstore" extension.
         value: ${{ jobs.slsa-run.outputs.attestations-download-name }}
 
+      provenance-download-sha256:
+        description: >
+          The sha256 digest of the attestations.
+
+          Users should verify the download against this digest to prevent tampering.
+        value: ${{ jobs.slsa-run.outputs.attestations-download-sha256 }}
+
+      binaries-download-name:
+        description: >
+          The name of the folder containing the built artifacts. There is a random hash at the
```

Comment:
the download name and the actual folder may be different. Which is it?

Reply:
name of the artifact to download (`name` argument to `actions/upload-artifact`) can be different from the actual folder argument `path` to the Action. Other builders have both outputs.

Are they always the same in your builder?

Reply:
Right. So, the input `binaries-download-name`, which is now `artifacts-download-name`, is the name to download but you get `folder.tgz` which you must tar right after.

After you extract the contents, the user gets the folder `bazel_builder_binaries_to_upload_to_gh_7bc972367cb286b7f36ab4457f06e369`.

I will add output for this string.

Reply:
Added.